Client/ContentBlocker/ContentBlocker+Safelist.swift

```diff
@@ -80,12 +85,27 @@ extension ContentBlocker {
     }
 
     func readSafelistFile() -> [String]? {
-        guard let fileURL = safelistFileURL() else { return nil }
+        if  let fileURL = safelistFileURL(),
+            FileManager.default.fileExists(atPath: fileURL.path) {
+            return readSafelistFile(fileURL: fileURL)
+
+            // read file data, then move to new location
+        } else if let oldFileURL = oldSafelistFileURL(),
+                  FileManager.default.fileExists(atPath: oldFileURL.path) {
+            let data = readSafelistFile(fileURL: oldFileURL)
+            if let newFileURL = safelistFileURL() {
+                try? FileManager.default.moveItem(at: oldFileURL, to: newFileURL)
```

Comment:
We could catch the error here and log it, wit ha message that says something like "failed to move file" or something descriptive like that.

Reply:
I have added a log, please review it.